addons/purchase_requisition/models/purchase_requisition.py

```diff
@@ -3,57 +3,33 @@
 
-
-class PurchaseRequisitionType(models.Model):
-    _name = "purchase.requisition.type"
-    _description = "Purchase Requisition Type"
-    _order = "sequence"
-
-    name = fields.Char(string='Agreement Type', required=True, translate=True)
-    sequence = fields.Integer(default=1)
-    exclusive = fields.Selection([
-        ('exclusive', 'Select only one RFQ (exclusive)'), ('multiple', 'Select multiple RFQ (non-exclusive)')],
-        string='Agreement Selection Type', required=True, default='multiple',
-            help="""Select only one RFQ (exclusive):  when a purchase order is confirmed, cancel the remaining purchase order.\n
-                    Select multiple RFQ (non-exclusive): allows multiple purchase orders. On confirmation of a purchase order it does not cancel the remaining orders""")
-    quantity_copy = fields.Selection([
-        ('copy', 'Use quantities of agreement'), ('none', 'Set quantities manually')],
-        string='Quantities', required=True, default='none')
-    line_copy = fields.Selection([
-        ('copy', 'Use lines of agreement'), ('none', 'Do not create RfQ lines automatically')],
-        string='Lines', required=True, default='copy')
-    active = fields.Boolean(default=True, help="Set active to false to hide the Purchase Agreement Types without removing it.")
-
-
 class PurchaseRequisition(models.Model):
     _name = "purchase.requisition"
     _description = "Purchase Requisition"
     _inherit = ['mail.thread', 'mail.activity.mixin']
     _order = "id desc"
 
-    def _get_type_id(self):
-        return self.env['purchase.requisition.type'].search([], limit=1)
-
-    name = fields.Char(string='Reference', required=True, copy=False, default='New', readonly=True)
-    origin = fields.Char(string='Source Document')
+    name = fields.Char(
+        string='Agreement', copy=False, readonly=True, required=True,
+        default=lambda self: _('New'), tracking=True)
+    reference = fields.Char(string='Reference')
     order_count = fields.Integer(compute='_compute_orders_number', string='Number of Orders')
     vendor_id = fields.Many2one('res.partner', string="Vendor", domain="['|', ('company_id', '=', False), ('company_id', '=', company_id)]")
-    type_id = fields.Many2one('purchase.requisition.type', string="Agreement Type", required=True, default=_get_type_id)
-    ordering_date = fields.Date(string="Ordering Date", tracking=True)
-    date_end = fields.Datetime(string='Agreement Deadline', tracking=True)
-    schedule_date = fields.Date(string='Delivery Date', index=True, help="The expected and scheduled delivery date where all the products are received", tracking=True)
+    requisition_type = fields.Selection([
+        ('blanket_order', 'Blanket Order'), ('purchase_template', 'Purchase Template')],
+         string='Agreement Type', required=True, default='blanket_order')
+    date_start = fields.Date(string="Start Date", tracking=True)
+    date_end = fields.Date(string='End Date', tracking=True)
```

Comment:
@ticodoo  change of Datetime to date is done after @thomasdeleval   and @ansh-odoo  suggestion.

Reply:
I didn't see it in the spec :thinking: and it's a potential loss of user data so I disagree with the change. If they really want it then OK, but I'm not sure if you need to include something in the upgrade script since you're changing the field type for an existing field